zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -71,7 +70,7 @@
   Gson gson = new Gson();
   private Notebook notebook;
   private NotebookServer notebookServer;
-  private SearchService notebookIndex;
+  private SearchService noteIndex;
```

Comment:
Term `noteIndex` is little bit confusing for me. Since you are working on naming, how about changing it into `noteSearchService`?